### DISCUSSION TITLE
SdFat with SPI1

### DISCUSSION THREAD

**Discussion by f5exo** [earlephilhower/arduino-pico#1890] [Q&A]
Hello,

Is there a better solution to use SdFat and SPI1.
Currently I do it like this:
I add in the SdFatConfig.h file of the ESP8266SdFat library:
#define SDCARD_SPI SPI1
#define SDCARD_SS_PIN 13
#define SDCARD_MOSI_PIN 15
#define SDCARD_MISO_PIN 12
#define SDCARD_SCK_PIN 14

This works, but I think it's not too academic.
And must be updated with each new release.

Thank you.

Erick

**Comment by earlephilhower** [comment]
It's explained here:  https://github.com/earlephilhower/arduino-pico/blob/master/docs/fs.rst#using-second-spi-port-for-sd

**Comment by f5exo** [comment]
It's good !

Many  thanks.
